test/sql/function/list/list_reverse.test

```diff
@@ -47,8 +47,10 @@ execute q1([5, 42, 3]);
 ----
 [3, 42, 5]
 
+statement ok
+create or replace table tbl_big as select range(5000) as list;
```

Comment:
This might be more of a limitation of the python sqllogictest runner rather than a malformed test, it did not like that these two statement were both part of the `query` test-statement.